### PR TITLE
Issue 68(69) after a clearn start from latest integration branch

### LINKED ISSUE #68: Clarify information from download file ino usage statistics page
This change requires:

- [ ] Change name of the downloaded file
- [ ] Change the column name in the file content

On the usage statistics page, when downloading the file for "No. plans during last year", the file name is "completed_plans.csv".
This is is misleading/confusing.

The name should be changed to "created_plans.csv"

The file content also has a column called "Completed Plans". This should be changed to "Created Plans"

### PR DESCRIPTION
**Pull request by pengyin-shan**
Fixes #68 #69  .

Changes proposed in this PR:
- #68 issue
- Also fixed rspec test to pass. 
- Start from a cleaned integration branch to remove some unexpected local changes

- Robcop issue stays. We will discuss this

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -8,4 +8,4 @@ MYSQL_PASSWORD=roadmap_password
 MYSQL_ROOT_PASSWORD=secret_password
 DEVISE_PEPPER="change_me"
 TRANSLATION_IO_API_KEY='change_me'
-WICKED_PDF_PATH="change_me"
+WICKED_PDF_PATH="/usr/local/bin/wkhtmltopdf"
```
Comment (lagoan) [inline .env:11]: This looks good as it will give an example of what we are looking for!

File: app/controllers/usage_controller.rb [hunk 2]
```diff
@@ -116,14 +116,14 @@ def yearly_plans
...
-      csv << [_("Month"), _("No. Completed Plans")]
+      csv << [_("Month"), _("No. Created Plans")]
       total = 0
       @plans_per_month.each do |data|
         csv << [data.date.strftime("%b-%y"), data.count]
         total += data.count
       end
       csv << [_("Total"), total]
-    end, filename: "completed_plans.csv")
+    end, filename: "created_plans.csv")
```
Comment (lagoan) [inline app/controllers/usage_controller.rb:126]: This is the meat of the issue, great speed :D

File: app/views/usage/_total_usage.html.erb [hunk 3]
```diff
@@ -31,7 +31,7 @@
       <%= select_tag "csv-field-sep", options_for_select(separators, separators[0]), {class: "single-char-select"} %>
     </form>
   </div>
-  <% if current_user.can_super_admin? %>
+  <% if current_user.can_super_admin? || current_user.can_org_admin? %>
```
Comment (lagoan) [inline app/views/usage/_total_usage.html.erb:34]: I think this line would be granting access to organizational administrators to all of DMP Assistant statistics. This is something that we want to look at in the future but for the time being, org admins should only be able to see their organizational stats.

File: app/views/usage/index.html.erb [hunk 4]
```diff
@@ -10,7 +10,7 @@
            locals: { user_count: @total_org_users,
                      plan_count: @total_org_plans,
                      separators: @separators } %>
-<% if @funder.present? %>
+<% if !current_user.can_org_admin? && !current_user.can_super_admin? %>
   <%= render partial: 'usage/template_statistics' %>
 <% else %>
```
Comment (lagoan) [inline app/views/usage/index.html.erb:15]: I think this piece of code would also allow org admins to see the link to the template statistics from all the system. If that is the case then we should avoid it.

You can look at the partials `template_statistics` nad `plans_by_template_chart`

The last template uses the method `prep_data_for_template_plans_chart` .

File: spec/controllers/usage_controller_spec.rb [hunk 5]
```diff
@@ -92,7 +92,7 @@
       get :yearly_plans
     end
     it "assigns the correct csv data" do
-      expected = "Month,No. Completed Plans\n" \
+      expected = "Month,No. Created Plans\n" \
```
Comment (lagoan) [inline spec/controllers/usage_controller_spec.rb:95]: Nice catch!

File: config/database.yml [hunk 6]
```diff
@@ -2,6 +2,10 @@ defaults: &defaults
   adapter: mysql2
   encoding: utf8mb4
   pool: 16
+  database: <%= ENV["MYSQL_DATABASE"] %>
+  username: <%= ENV["MYSQL_USER"] %>
+  password: <%= ENV["MYSQL_PASSWORD"] %>
+  host: <%= ENV["DATABASE_URL"] || '127.0.0.1' %>
```
Comment (lagoan) [inline config/database.yml:8]: It is odd that the previous code did not work for you. Maybe it is a Mac thing? We can leave it as is for now.

**Review by lagoan — CHANGES_REQUESTED** [review]
Thinks look really good. I just have some comments on who can access  usage statistics.

File: config/database.yml (hunk 6 above)
Comment (pengyin-shan) [reply config/database.yml:8]: I'll skip this file in the future when pushing to remote repository. It should be just a configuration thing so let's use your version in the integration branch.

**Comment by pengyin-shan** [comment]
@lagoan  just to clarify, the 'download' button you mean for **org admin** is one of the following (there are three download buttons for user statistics):
Screen Shot 2021-12-01 at 2 40 29 PM
or the 'global download usage' button on the most top? 
or a new download button for template statistics like this
Screen Shot 2021-12-01 at 2 43 55 PM
:

**Review by lagoan — APPROVED** [review]

**Pull request merged**